web/client/plugins/FeatureEditor.jsx

```diff
@@ -47,10 +47,41 @@ import {isViewportFilterActive} from "../selectors/featuregrid";
   *        "editorProps": {
   *            "values": ["Option1", "Option2", "Option3", "Option4"]
   *        }
-  *    }
+  *    }, {
+  *        "regex": {
+  *            "attribute": "SUB_REGION",
+  *            "typeName": "gs:us_states"
+  *        },
+  *        "editor": "CustomAutocompleteEditor",
+  *        "editorProps": {
+  *            "filterProps": {
+  *                "blacklist": [],
+  *                "maxFeatures": 3,
+  *                "queriableAttributes": ["FIELD01"],
+  *                "predicate": "ILIKE",
+  *                "typeName": "test:layer01",
+  *                "srsName": "EPSG:4326",
+  *                "performFetch": true
+  *            }
+  *        }
   *    }]
   *}
   * ```
+  *
+  * ### For editor `CustomAutocompleteEditor` in cfg.rules
+  * The `CustomAutocompleteEditor` enables dynamic fetching of unique attribute values from a source layer to provide
+  * autocomplete functionality when editing attributes in a target layer. This is particularly useful for maintaining
+  * data consistency and providing users with valid options based on existing data from related layers.
```

Comment:
```suggestion
```
Remove it from the plugin doc and add it to [customEditors](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/components/data/featuregrid/editors/customEditors.jsx)